config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-postsubmits.yaml

```diff
@@ -21,7 +22,15 @@ postsubmits:
             secretKeyRef:
               name: nfd-creds
               key: codecov-token
+        resources:
+          limits:
+            cpu: 4
+            memory: 8Gi
+          requests:
+            cpu: 4
+            memory: 8Gi        
   - name: postsubmit-node-feature-discovery-e2e-test
+    cluster: eks-prow-build-cluster
```

Comment:
if you're doing GCP e2e tests you should use the GCP cluster (k8s-infra-prow-build)

Reply:
(other jobs like unit test / verify, AWS e2e can and should use the EKS cluster to spread the load between the cloud credits)